jsapp/js/components/submissionDataTable.es6

```diff
@@ -124,24 +125,34 @@ class SubmissionDataTable extends React.Component {
       case QUESTION_TYPES.get('select_one').id:
       case SCORE_ROW_TYPE:
       case RANK_LEVEL_TYPE:
-        choice = this.findChoice(data);
-        return (
-          <bem.SubmissionDataTable__value>
-            {choice.label[this.props.translationIndex]}
-          </bem.SubmissionDataTable__value>
-        );
+        choice = this.findChoice(listName, data);
+        if (!choice) {
+          console.error(`Choice not found for "${listName}" and "${data}".`);
+          return null;
+        } else {
+          return (
+            <bem.SubmissionDataTable__value>
+              {choice.label[this.props.translationIndex] || choice.name}
+            </bem.SubmissionDataTable__value>
+          );
+        }
       case QUESTION_TYPES.get('select_multiple').id:
         return (
           <ul>
             {data.split(' ').map((answer, answerIndex) => {
-              choice = this.findChoice(answer);
-              return (
-                <li key={answerIndex}>
-                  <bem.SubmissionDataTable__value>
-                    {choice.label[this.props.translationIndex]}
-                  </bem.SubmissionDataTable__value>
-                </li>
-              );
+              choice = this.findChoice(listName, answer);
+              if (!choice) {
+                console.error(`Choice not found for "${listName}" and "${data}".`);
+                return null;
```

Comment:
pasting comment from above (substitute `answer` for data)
> the `|| choice.name` below is good to deal with untranslated labels, but sometimes the choice may disappear from the latest version entirely. in that case, i think `data` should be shown instead of nothing :)

Reply:
Makes sense, fixed :+1: